### PR TITLE
Update announcements.json

### PR DESCRIPTION
**Pull request by TamGIBill**
Add two announcements for SAA and SCO Office Hours

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/static-pages/school-resources/constants/announcements.json [hunk 1]
```diff
@@ -1,4 +1,19 @@
 [
+{
+    "name": "Webinar on December 11 – Office Hours – For State Approving Agencies (SAAs)",
+    "url": "https://benefits.va.gov/gibill/index.asp#20191205",
+    "date": "2019-12-05",
+    "displayStartDate": "2019-12-05"
```
Comment (shawkey-daniel) [inline src/applications/static-pages/school-resources/constants/announcements.json]: This line needs a comma at the end

File: src/applications/static-pages/school-resources/constants/announcements.json [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+    "displayEndDate": "2019-12-13"
+  },
+  
+  {
+    "name": "Webinar on December 12 – Office Hours – For State Certifying Officials (SCOs)",
+    "url": "https://benefits.va.gov/gibill/index.asp#201912052",
+    "date": "2019-12-05",
+    "displayStartDate": "2019-12-05"
```
Comment (shawkey-daniel) [inline src/applications/static-pages/school-resources/constants/announcements.json]: This line also needs a comma at the end

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

**Review by rianfowler — APPROVED** [review]

**Review by rianfowler — APPROVED** [review]

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Comment by shawkey-daniel** [comment]
Created #11288 to get changes in, since this one will not pass e2e tests in Jenkins.

**Pull request closed without merging**